src/profiles_management/create_or_update.py

```diff
@@ -73,11 +73,24 @@ def create_or_update_profiles(client: Client, pmr: ProfilesManagementRepresentat
     log.info("Fetching all Profiles in the cluster")
 
     existing_profiles: Dict[str, GenericGlobalResource] = {}
-    for profile in list_profiles(client):
+    for profile in profiles.list_profiles(client):
         existing_profiles[get_name(profile)] = profile
 
-    log.info("Removing access to all stale Profiles")
+    log.info("Removing access to all stale Profiles.")
```

Comment:
suggestion(non-blocking): Maybe add a comment on what this section does, as you do below
```suggestion
    # Remove access to Profiles that are considered stale
    log.info("Removing access to all stale Profiles.")
```